src/main/java/net/gensokyoreimagined/motoori/Kosuzu.java

```diff
@@ -59,6 +59,8 @@ public void onEnable() {
 
         config.addDefault("match.include", regexDefaults);
 
+        config.addDefault("match.blacklist", List.<String>of());
```

Comment:
Please use `Collections.emptyList()` for semantic reasons - I have never seen anyone use this for an emtpy list.

Reply:
Oh, and add the `java.util.Collections` import if necessary.